corehq/apps/domain/views/settings.py

```diff
@@ -271,6 +273,71 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class EditIPAccessConfigView(BaseProjectSettingsView):
+    template_name = 'domain/admin/ip_access_config.html'
+    urlname = 'ip_access_config'
+    page_title = gettext_lazy("IP Access")
+
+    @method_decorator(always_allow_project_access)
+    @method_decorator(login_and_domain_required)
+    @use_bootstrap5
+    def dispatch(self, *args, **kwargs):
+        if IP_ACCESS_CONTROLS.enabled(self.domain):
+            return super().dispatch(*args, **kwargs)
+        else:
+            raise Http404()
+
+    @property
+    @memoized
+    def form(self):
+        initial = {}
+        domain_config = self.get_ip_access_config
+        if domain_config:
+            display_spacer = ", "
+            initial.update({
+                'country_allowlist': domain_config.country_allowlist,
+                'ip_allowlist': display_spacer.join(domain_config.ip_allowlist),
+                'ip_denylist': display_spacer.join(domain_config.ip_denylist),
+                'comment': domain_config.comment
+            })
+
+        if self.request.method == 'POST':
+            return IPAccessConfigForm(self.request.POST, initial=initial)
+        return IPAccessConfigForm(initial=initial)
+
+    @property
+    def get_ip_access_config(self):
```

Comment:
Nit: I'd either make this a `@cached_property` and rename it `ip_access_config` or drop `@property`.  It's surprising to see something called `get_*` that you don't call with parenthesis.